packages/app/src/app/pages/Dashboard/Content/routes/Recent/DocumentationRow.tsx

```diff
@@ -7,6 +8,13 @@ type VideoProps = React.ComponentProps<typeof VideoCard>;
 
 type DocsItem = { label: string } & (VideoProps | ArticleProps);
 
+export const appendOnboardingTracking = (url: string): string => {
```

Comment:
This is almost like the "add stripe search params" function I created. Whenever both are merge we could look into creating a general util for it maybe :)

Reply:
@tristandubbeld do you think we should have a generic `appendParams`? It can take a single object, a list of objects, etc?